monitoring/volume_collector.go

```diff
@@ -1,6 +1,8 @@
 package monitoring
 
 import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
```

Comment:
Wrong location for import.

Reply:
Sorry, I forgot the imports statements again. Fixed it